modules/react/popup/stories/Popup.stories.mdx

```diff
@@ -0,0 +1,318 @@
+import {RTL} from './examples/RTL';
+import {UsePopup} from './examples/UsePopup';
+import {PopupModelConfigComponent, PopupStateComponent, PopupEventsComponent} from './PopupModel';
+
+<Meta title="Components/Popups/Popup/React" component={Popup} />
+
+# Canvas Kit Popups
+
+A "popup" is a classification for a type of stacked UI element that appears "on top" of statically
+positioned content. Tooltips, Modals, Dropdown menus, etc are all examples of "popups". Canvas Kit
+has a "stack manager" system for managing these stacked UIs. Different types of popups have
+different requirements of behavior for UX and accessibility - we can call them behaviors,
+capabilities, or traits. Canvas Kit comes with a number of [behaviors](#hooks) in the form of React
+Hooks.
+
+You should use the most semantic component for your use-case before using `Popup` directly, like
+`Modal`, which already has the correct behaviors built-in. If no component already exists, you can
+use `Popup` and use our [hooks](#hooks). The `Popup` component comes with a `Popup.Popper` component
+that positions a popup using [PopperJS](https://popper.js.org/) that registers a popup with the
+`PopupStack` automatically and sets the popup model's `placement` property. `Popup.Popper` component
+and hooks work with the stack management system for correct rendering and accessibility behavior. If
+you cannot use `Popup.Popper`, use the [usePopupStack](#usepoupstack) hook to properly register and
+deregister the popup at the correct time. If you cannot use our hooks, consider upgrading your
+component to use Hooks. If you cannot do that, you'll have to look up the `PopupStack` package for
+the direct API and have a look at the source code for our hooks into the `PopupStack` API.
+
+This package comes with everything you need to build Popup UIs.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+The `Popup` component is a low-level compound component that is used to build popup UIs that are not
+already covered by Canvas Kit.
+
+### Basic
+
+The Popup has no pre-defined behaviors already registered, therefore the `usePopupModel` must always
+be used to create a new `model`. This `model` is then used by all behavior hooks to apply additional
+popup behaviors to the compound component group. The following example creates a typical popup
+around a target element and adds `useCloseOnOutsideClick`, `useCloseOnEscape`, `useInitialFocus`,
+and `useReturnFocus` behaviors. You can read through the [hooks](#hooks) section to learn about all
+the popup behaviors. For accessibility, these behaviors should be included most of the time.
+
+<ExampleCodeBlock code={Simple} />
+
+### Without a model
+
+This example shows how a `Popup.Card` can be used outside a `Popup` component, but accessibility is
+lost. If you use the Popup this way, you'll have to manually add a `aria-labelledby` to the
+`Popup.Card` component and an `id` to the `Popup.Heading` component.
```

Comment:
When/why would you want/need to do this?

Reply:
I don't think people should do this, but it is the easiest way to upgrade. I'm not sure where this type of transition documentation should be, but it will be questions we'll commonly get. I wouldn't want v5 adoption to be blocked by Popups being too difficult to migrate to for some teams.

Reply:
Maybe we put that in the migration guide rather than in our storybook docs then?